ENIGMAsystem/SHELL/Platforms/SDL/Window.h

```diff
@@ -14,6 +14,7 @@ namespace enigma
     extern std::unordered_map<SDL_Keycode,int> keymap;
     extern std::unordered_map<SDL_Keycode, int> inverse_keymap;
   }
+  void window_id_init();
```

Comment:
is this sdl specific? I feel like it should be declared in general

Reply:
I believe it's specific to subsystems of SDL. Other systems only have one way of implementing their window handle logic, because it happens naturally while creating the window, but SDL needs to pretend to be each other operating system itself.

Reply:
It's declaired on the other platforms a different way iirc. I did this for SDL so that window_handle could be captured differently per platform